scripts/macos/utils/commons.sh

```diff
@@ -94,6 +94,10 @@ utest_cleanup_nativemessaging() {
   _cleanup tests.xcodeproj/ ./Release/tests || die
 }
 
+utest_grcov() {    
+  grcov .obj/tests.build/Release/tests.build/Objects-normal/x86_64/  -s . -t lcov --branch --ignore-not-existing . -o "$1" || die "Failure running grcov unit"
```

Comment:
I still think having a `_grcov` function is nicer. In here you have a lot of dup code (all these params).
If you implement a simple:
```
_grcov {
  grcov "$2"  -s . -t lcov --branch --ignore-not-existing . -o "$1" || die "Failed to run grcov"
}
```

and then here you run:

`_grcov .obj/tests.build/Release/tests.build/Objects-normal/x86_64/ "$1"`